src/MainPage.js

```diff
@@ -22,7 +22,7 @@ function MainPage({
                 errorMessage={errorMessage}
                 isDataLoading={isDataLoading}
                 setSearchOrigin={setSearchOrigin}
-            ></Form>
```

Comment:
Could be there a more descriptive name for the form and what it is doing? Hard to think of something, but maybe you have a practical idea. ``CheckHotspotForm``or something like this?